### PR TITLE
Parameterize schema for pg extension (uuid-ossp)

### PR DESCRIPTION
**Pull request by fantix**
This supports the latest Heroku where extensions must be created under the schema "heroku_ext" instead of our preferred "edgedbext".

### CODE REVIEW

File: edb/pgsql/compiler/config.py [hunk 1]
```diff
@@ -412,7 +412,10 @@ def _rewrite_config_insert(
 
     overwrite_query = pgast.SelectStmt()
     id_expr = pgast.FuncCall(
-        name=('edgedbext', 'uuid_generate_v1mc',),
+        name=(
+            ctx.env.backend_runtime_params.instance_params.ext_schema,
```
Comment (elprans) [inline edb/pgsql/compiler/config.py:416]: No need to use `instance_params` here, just use the wrapper in `edgedb` here.

**Comment by fantix** [comment]
To get a Heroku Postgres at version 14:

```
heroku addons:create -a app_name heroku-postgresql:basic --version 14
```

File: edb/pgsql/metaschema.py [hunk 2]
```diff
@@ -4409,9 +4409,54 @@ def __init__(self) -> None:
         )
 
 
+class UuidGenerateV1mcFunction(dbops.Function):
```
Comment (msullivan) [inline edb/pgsql/metaschema.py:4412]: Is there any chance these extra wrappers will slow things down? Probably they can get inlined?
(Is preventing that why they are marked parallel_safe?)

**Review by msullivan — APPROVED** [review]
Great

File: edb/pgsql/metaschema.py (hunk 2 above)
Comment (elprans) [reply edb/pgsql/metaschema.py:4412]: They should get inlined.  See criteria for inlining [here](https://github.com/postgres/postgres/blob/919c486a275bcdd83d2add77da87b6edbd91418c/src/backend/optimizer/util/clauses.c#L4401)

>  (Is preventing that why they are marked parallel_safe?)

No, the underlying function is marked as parallel-safe, so the wrapper should also.

**Pull request merged**